app/src/main/java/com/example/vehicleinfocheck/WebActivity.java

```diff
@@ -19,28 +18,28 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_web);
-        setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_SENSOR_PORTRAIT); // Fixes orientation to PORTRAIT mode
+        setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_SENSOR_PORTRAIT);   // Fixes orientation to PORTRAIT mode
         EditText vehicleNumber = findViewById(R.id.VehicleNum);
         ImageButton CopyButton = findViewById(R.id.CopyBtn);
         LinearLayout webview= findViewById(R.id.Webview);
         WebScraper webScraper = new WebScraper(this);   // Creating an instance of the WebScraper class
 
         webScraper.setUserAgentToDesktop(true);                 // Default: false
         webScraper.setLoadImages(true);                         // Default: false
-        webview.addView(webScraper.getView());                  //Setting the LinearLayout to display the contents of the webScraper
-        String FULL_URL = "https://vahan.nic.in/nrservices/faces/user/searchstatus.xhtml";
-        webScraper.loadURL(FULL_URL);
+        webview.addView(webScraper.getView());                  // Setting the LinearLayout to display the contents of the webScraper
+        String FullUrl = "https://vahan.nic.in/nrservices/faces/user/searchstatus.xhtml";
+        webScraper.loadURL(FullUrl);
         Toast.makeText(this, "Loading website...", Toast.LENGTH_SHORT).show();
 
         // OnClickListener triggered when CopyButton is clicked
         CopyButton.setOnClickListener(v -> {
             // Creating an instance of ClipboardManager class
             ClipboardManager clipboard = (ClipboardManager) getSystemService(Context.CLIPBOARD_SERVICE);
-            ClipData clip = ClipData.newPlainText("simple text", vehicleNumber.getText());    // Creates a clip with value = vehicleNumber
-            clipboard.setPrimaryClip(clip);
-            if (!vehicleNumber.getText().toString().trim().equals("")) {
-                Toast.makeText(this, "Copied to clipboard", Toast.LENGTH_SHORT).show();// Displayed only if there is text to copy
+            if (!vehicleNumber.getText().toString().trim().equals("")) {                        // Clip is made only if there is text to copy
```

Comment:
Shouldn't there be a condition to check if the text is empty and throw an error message?